packages/contracts/tests/Poll.test.ts

```diff
@@ -96,5 +96,30 @@ describe("Poll", () => {
       );
       await expect(pollContract.init()).to.be.revertedWithCustomError(pollContract, "PollAlreadyInit");
     });
+
+    it("should not be possible to set zero address as registry manager", async () => {
```

Comment:
should these go to their own Maci.test.ts file?

Reply:
I think this file can be renamed to Maci.test.ts because most of the tests are related to MACI contract here.